src/CacheWarmer/EntrypointCacheWarmer.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony WebpackEncoreBundle package.
+ * (c) Fabien Potencier <fabien@symfony.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\WebpackEncoreBundle\CacheWarmer;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpKernel\CacheWarmer\CacheWarmerInterface;
+use Symfony\WebpackEncoreBundle\Asset\EntrypointLookupCollection;
+
+class EntrypointCacheWarmer implements CacheWarmerInterface
+{
+    private $builds;
+    private $container;
+
+    public function __construct(array $builds, ContainerInterface $container)
+    {
+        $this->builds = $builds;
+        $this->container = $container;
+    }
+
+    public function isOptional()
+    {
+        return true;
+    }
+
+    public function warmUp($cacheDir)
+    {
+        $entryPointCollection = $this->container->get('webpack_encore.entrypoint_lookup_collection');
+
+        if ($entryPointCollection instanceof EntrypointLookupCollection) {
```

Comment:
We could simplify if we added a getAll() method to this object. Then we wouldn’t need to inject builds here. We could just foreach over getAll()

Reply:
Yea, I think we should actually inject the EntrypointLookupCollection service here directly - I think there’s no need to inject the container and then fetch it. Then, yes, the getAll() method would be on *that* class (the collection class) - it will allow us to easily ask for all the EntrypointLookup instances.

Reply:
The collection class gets an instance of ContainerInterface this does not have a get all. So should I pass the builds to the collection? So it can track them. When I debug through I see it is the ServiceLocator class from the dependence injection component. I dont see a get all method there but I can see how it works and try to extend it to make that possible.